test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,15 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // only one null object
+        assertTrue(Utils.isAnyNull((Object) null));
+        // only one object is null
+        assertTrue(Utils.isAnyNull("a", 1, false, true, null));
+        // no object is null
+        assertFalse(Utils.isAnyNull(1, "a", 1, false, true));
```

Comment:
Generally ok, but maybe you want more than a single test for a given scenario if possible.
And remember to consider empty argument as well.